lib/ansible/modules/windows/setup.ps1

```diff
@@ -307,7 +307,7 @@ if($gather_subset.Contains('memory')) {
 if($gather_subset.Contains('platform')) {
     $win32_cs = Get-LazyCimInstance Win32_ComputerSystem
     $win32_os = Get-LazyCimInstance Win32_OperatingSystem
-    $ip_props = [System.Net.NetworkInformation.IPGlobalProperties]::GetIPGlobalProperties()
+    #$ip_props = [System.Net.NetworkInformation.IPGlobalProperties]::GetIPGlobalProperties()
```

Comment:
No use keeping this commented, we have git history if we need to see what was originally there.